chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -9,6 +9,8 @@
     mon_init << monitor.name.split(".")[0]
 end
 
+# These directories should all be created by the package, maybe we can
+# just remove this stuff...?  (same with the directories created in osd.rb)
```

Comment:
It's unclear if they're needed for other distros, so one approach would be to only create them for non-SUSE platform families.